recipes/monet/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "monet" %}
+{% set version = "2.1.5" %}
+
+package:
+  name: monet
+  version: {{ version }}
+
+source:
+  #git_url: https://github.com/noaa-oar-arl/MONET.git
+  #git_tag: develop
+  url: https://github.com/noaa-oar-arl/MONET/archive/v2.2.0.tar.gz
+  sha256: 293ca8ca167c77843a165e6032f18ea50b96f964264087ce15dabde4bf0a270e
+  #url: https://github.com/noaa-oar-arl/MONET/archive/v2.1.3-dev.2.tar.gz
+  # sha256: 7027a0478cab0f877ebc7e412a0249d0578c0944fab4da6f341175995b957931
+
+build:
+  number: 0
+  #noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python >=3.6
```

Comment:
You don't need `python` listed under `build`. Is this package python3-only though?

Reply:
yes it is python 3 only